AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java

```diff
@@ -567,7 +567,7 @@ private void _addDummyCollection(ZipFile zip, Context context) throws IOExceptio
  * @author Tim
  */
 class ZipFile {
-    private final int BUFFER_SIZE = 1024;
+    private final int mM_BUFFER_SIZE = 1024;
```

Comment:
And revert the other changes to the fie
```suggestion
     private static final int BUFFER_SIZE = 1024;
```